docs/developers/developing_iree/profiling_with_tracy.md

```diff
@@ -1,15 +1,16 @@
 # Profiling with Tracy
 
 [Tracy](https://github.com/wolfpld/tracy) is a profiler that puts together in a
-single view:
-
-*   Both CPU and GPU profiling.
-*   Both sampling and instrumentation.
-*   Both specifics of our own process, and whole-system profiling a la
-    "systrace".
-
-Since Tracy relies on instrumentation, it requires IREE binaries to be built
-with a special flag to enable it.
+single view both instrumentation and system profiling (sampling, systrace). It's
+key to understand the nuance here.
+* *Instrumentation* is code built into the process being profiled, collecting
+  timestamps at the start and end of "zones". Once it's enabled at build time,
+  it typically just works &mdash; it is a part of our application logic just
+  like anything else, so there's no reason why it would not work.
+* *Sampling* and *SysTrace* rely on specific
+  system features to collect information on what is *actually* running. These
+  rely on OS and binary (ELF) file features, so they can take a bit more care to
+  get to work properly.
```

Comment:
Screenshots would be nice here (maybe hosted on github, to not bloat our repo side). Future work :D